### ISSUE TITLE
Change SmartSize to use `cx`/`cy`

### ISSUE THREAD

**Issue by rexxars** [imbo/imbo#378]
Right now we're storing points of interest, but going forward we might want to support an area of interest, which will need a width and a height.

When this is introduced, I think it'll be weird to have to calculate the start and end positions based on the center of this point. Instead, I think we should make the smartSize transformation use `cx` (center x) and `cy` (center y), and leave the `x` and `y` available for when we add "area of interest" support.

When this is added, I could see us having areas of interest with all of the following:

```js
{
  cy: 200,
  cx: 200,
  x: 100,
  y: 100,
  width: 300,
  height: 300
}
```

This would make it more consistent with our other transformations, such as `crop`, which takes `x`, `y`, `width` and `height`.

This is a breaking change, but since 2.0 is not released yet I think it would be a good idea to do this now rather than later.

**Comment by rexxars** [comment]
@kbrabrand Is it alright if I go ahead and implement this change, or do you have any objections?

**Comment by kbrabrand** [comment]
👍 No objections.

**Comment by rexxars** [comment]
Implemented in #382
